src/components/InfoPanel/infoPanel.module.scss

```diff
@@ -3,35 +3,43 @@
   font-weight: bold;
   background-color: white;
   display: flex;
+  flex-direction: row;
   justify-content: center;
   align-items: center;
   position: absolute;
   bottom: calc(-50% + 75px);
   right: calc(50% - 400px);
   border: 1px solid black;
   border-radius: 10px;
+  z-index: 3000;
```

Comment:
Why 3000?

Reply:
:) that question I forgot to ask in our meeting. I tried make 1,2,3,10 and it doesn't work. In your course you also used z-index like 99999
If 99999 better i can use it?

Reply:
It depends =)
We can proceed with you current solution and discuss this topic afterwards 👍